azavea/temperate
# Create plan community systems step

## Overview

Implement third step of 'create plan' wizard, where user selects community systems for their organization. Also adds community systems to organization model and serializer.

### Demo

None selected:
![image](https://user-images.githubusercontent.com/960264/36619091-8a2959c6-18bb-11e8-8885-87319ed7f6af.png)

With selections:
![image](https://user-images.githubusercontent.com/960264/36619145-bc1637e2-18bb-11e8-8343-5de272a80e65.png)


### Notes

Differs from very similar picker components on the previous hazards step in that selections can also be removed by toggling the list items, and that some default content displays in the top section if nothing has been selected. Both could easily be added to second step as well, if wanted.


## Testing Instructions

 * Run migrations
 * Unset user's primary organization, or create a new user, to get to 'create plan' wizard
 * Should be able to pick community systems on third step
 * Selected systems should save on organization
 * Can check saved system IDs at http://localhost:8100/api/user/

Closes #489
